lib/server.js

```diff
@@ -36,6 +35,8 @@ function Server(opts){
   this.clientsCount = 0;
 
   opts = opts || {};
+
+  this.wsEngine = opts.wsEngine || process.env.EIO_WS_ENGINE || 'ws';
```

Comment:
Isn't this the same problem again? The option itself can be 'wsEngine' but the (global to the process) should probably be named something implying it's only a server option, right? EIO_SERVER_WS_ENGINE but option wsEngine?